public/danangbus/js/main.js

```diff
@@ -0,0 +1,78 @@
+$(document).ready(function() {
+
+showBusStopOnMap();
+});
+function showBusStopOnMap() {
+  var busstops = JSON.parse(routeJSONStr);
```

Comment:
the `main.js` is used for many pages, but only route detail page have the variable `routeJSONStr`. So another pages will have errors.